FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice/covg_multiple_priorities_no_timeslice_utest.c

```diff
@@ -655,7 +660,6 @@ void test_coverage_prvAddNewTaskToReadyList_create_two_tasks_with_the_first_susp
     vTaskStartScheduler();
 }
 
-#if 0
 /**
  * @brief prvAddNewTaskToReadyList - add a new idle task to the list of ready tasks
```

Comment:
Just as a suggestion, it might be worth mentioning in the comments that there are two prvAddNewTaskToReadyList() functions and this test case applies to the one where configNUMBER_OF_CORES > 1